src/SFA.DAS.EmployerDemand/InnerApi/Requests/GetCourseProviderDemandsRequest.cs

```diff
@@ -1,4 +1,8 @@
-﻿using SFA.DAS.SharedOuterApi.Interfaces;
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Web;
```

Comment:
plz check if these using are needed, doesn't seem to be any other change to this file.